config/build/webpack.config.common.js

```diff
@@ -34,7 +35,7 @@ export const createResolveConfigLoader = () => ({
 
 export function config(options) {
     return {
-        devtool: options.dev && 'source-map',
+        devtool: options.dev && 'eval-cheap-module-source-map',
```

Comment:
Aww :(
You have the link to the corresponding issue ? We could at least link it to here / upvote it.